src/colecaoLivro.py

```diff
@@ -0,0 +1,14 @@
+from src.livro import Livro
+
+class ColecaoDeLivro:
+    def __init__(self):
+        self.__lista = []
+        
+    def incluir(self, livro: Livro):
+        if livro in self.__lista:
+            raise Exception("Livro com mesmo titulo ja existe")
```

Comment:
Sua validação é de título e isbn, por que a mensagem considera só o título. Aqui, entra o caso de se for específicar o que pode dar errado, deixe todos os campos na mensagem. É tudo ou nada :)